### DISCUSSION TITLE
Welcome to Quran-Flutter Discussions!

### DISCUSSION THREAD

**Discussion by Sadmansamee** [SadaqaWorks/Quran-Flutter#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GibreelAbdullah** [comment]
Gibreel Abdullah, Data Engineer/ Flutter Developer.

Has this been published on the Play Store? And what is the future roadmap?

**Comment by Mohamed-7018** [comment]
I face the following error so could you please help me to fix it and thank you


E/flutter (10082): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: HiveError: Cannot write, unknown type: _$_Resource. Did you forget to register an adapter? E/flutter (10082): #0 BinaryWriterImpl.write (package:hive/src/binary/binary_writer_impl.dart:337:9) E/flutter (10082): #1 BinaryWriterImpl.writeList (package:hive/src/binary/binary_writer_impl.dart:223:7) E/flutter (10082): #2 BinaryWriterImpl._writeList (package:hive/src/binary/binary_writer_impl.dart:389:7) E/flutter (10082): #3 BinaryWriterImpl.write (package:hive/src/binary/binary_writer_impl.dart:328:7) E/flutter (10082): #4 BinaryWriterImpl.writeFrame (package:hive/src/binary/binary_writer_impl.dart:281:9) E/flutter (10082): #5 StorageBackendVm.writeFrames.<anonymous closure> (package:hive/src/backend/vm/storage_backend_vm.dart:127:31) E/flutter (10082): #6 StorageBackendVm.writeFrames.<anonymous closure> (package:hive/src/backend/vm/storage_backend_vm.dart:123:28) E/flutter (10082): #7 ReadWriteSync.syncWrite.<anonymous closure> (package:hive/src/backend/vm/read_write_sync.dart:26:41) E/flutter (10082): #8 _rootRunUnary (dart:async/zone.dart:1436:47) E/flutter (10082): #9 _CustomZone.runUnary (dart:async/zone.dart:1335:19) E/flutter (10082): <asynchronous suspension> E/flutter (10082): #10 BoxImpl._writeFrames (package:hive/src/box/box_impl.dart:88:7) E/flutter (10082): <asynchronous suspension> E/flutter (10082):
